src/components/custom-button/tailwind-button.tsx

```diff
@@ -0,0 +1,56 @@
+import React from 'react';
+import {
+  ActivityIndicator,
+  Text,
+  TouchableOpacity,
+  TouchableOpacityProps,
+} from 'react-native';
+
+const ButtonContainer = TouchableOpacity;
+
+interface TailwindButtonProps extends TouchableOpacityProps {
+  loading?: boolean;
+  title: string;
+  variant?: 'primary' | 'secondary';
+}
+
+const TailwindButton: React.FC<TailwindButtonProps> = ({
+  loading,
+  title,
+  variant = 'primary',
+  ...props
+}) => {
+  const buttonStyle = [
+    'px-2.5',
+    'py-1.5',
```

Comment:
This fails the purpose of adding a Component library, I don't think this would work. 
There are places in the components from react itself that have the props `contentContainerStyle`, `headerStyle` and all like in Flatlist. how are we going to implement in those.
and if there is need for some new components like google maps, I don't think you can do this.
Please try to think about those scenarios and come up with something robust.
I'll suggest you to look at some open source apps created with NativeWind and see what they do to mitigate the above issues.

Reply:
Hey,
Looked at multiple open source apps with `nativewind` integration and found out that these styles are not handled with tailwind instead it is used normally with either stylesheet or inline styles.
The best alternative I could find to apply tailwind CSS is using [twrnc](https://www.npmjs.com/package/twrnc?activeTab=readme).
It can be used with most of the style props using tailwind CSS.